Wikipedia/Categories/MWKSection+DisplayHtml.m

```diff
@@ -22,13 +22,20 @@ - (NSString*)displayHTML:(NSString*)html {
 }
 
```

Comment:
Instead of just using *<H1>* tags for all section headers, use the full range of *H1* through *H6* according to level depth.